src/args.ts

```diff
@@ -69,4 +69,34 @@ export let args : ArgDescriptor[] = [
       ' Defaults to "index.html".',
     type: String,
   },
+  {
+    name: 'protocol',
+    alias: 'h',
+    description: 'The server protocol to use {h2, https/1.1, http/1.1}.' +
+      ' Defaults to "http/1.1".',
+    defaultValue: 'http/1.1',
+    type: String,
+  },
+  {
+    name: 'key',
+    alias: 'k',
+    description: 'Path to TLS certificate private key file for https.' +
+      ' Defaults to "key.pem".',
+    defaultValue: 'key.pem',
+    type: String,
+  },
+  {
+    name: 'cert',
+    alias: 't',
```

Comment:
given we share aliases with the CLI by default, how about leaving off alias for options with decent defaults. I'd leave off all but protocol.